WebCompiler/Compile/Compilers.cs

```diff
@@ -14,12 +14,14 @@ public class Compilers
         private readonly Compiler zipper;
         private readonly Compiler place;
         private readonly Compiler cleanup;
+        private readonly Compiler autoprefix;
 
         public Compilers(Config config, string base_path)
         {
-            sass = new SassCompiler(config.CompilerSettings.Sass);
+            sass = config.Autoprefix.Enabled ? new SassCompiler(config.CompilerSettings.Sass, config.Autoprefix) : new SassCompiler(config.CompilerSettings.Sass);
```

Comment:
I'm not quite sure I follow why this is necessary. Shouldn't we just add the `autoprefix` step to line 36? Then the `sass` compiler would not need to be made more complex, wouldn't it?

Reply:
Autoprefixer in this case uses the source map from the output of the sass compilation.

Another approach would be to output the sourcemap files and pull from those but I thought this would be simpler.